README.md

```diff
@@ -1,3 +1,47 @@
-# Tic-Tac-Toe-Js
-This is a Tic Tac Toe game built with JavaScript that can be played in the browser
-By Odin Project
\ No newline at end of file
+# Tic Tac Toe Project - [Microverse - JavaScript Course]
+
+> ​ We’re making a Tic Tac Toe game you can play in your browser!.
+
+![screenshot](TicTacToe.png )
+
+Additional description about the project and its features.
+
+## Link to Project
+[OdinProject](https://www.theodinproject.com/courses/javascript/lessons/tic-tac-toe-javascript)
+
+## Built With
+
+- HTML/CSS,
+- Javascript
+
+## Live Demo
+
+[Live Demo Link](https://rawcdn.githack.com/damiecode/Tic-Tac-Toe-Js/feature/Tic-Tac-Toe/index.html )
```

Comment:
The live link may become broken if the `feature` branch is deleted.

You could use `https://rawcdn.githack.com/damiecode/Tic-Tac-Toe-Js/feature/Tic-Tac-Toe/index.html` instead or a commit specific URL like `https://rawcdn.githack.com/damiecode/Tic-Tac-Toe-Js/ae91a6112a67103254fe556cf2b11d5712f1b1b1/index.html`

Reply:
Thank you @bolah2009